packages/imperative/src/config/cmd/init/init.definition.ts

```diff
@@ -76,6 +92,14 @@ export const initDefinition: ICommandDefinition = {
         {
             description: "Do a dry run of initializing the user config files and do not prompt for secure values.",
             options: "--user-config --prompt false --dry-run"
+        },
+        {
+            description: "Overwrite any existing global config files",
+            options: "--global-config --overwrite"
```

Comment:
Should this, and the example below, also include the --for-sure flag?

Reply:
Yes, I added it for both examples. Thanks for pointing it out.